### DISCUSSION TITLE
Github Classroom issue connecting to JS Practice 3

### DISCUSSION THREAD

**Discussion by fimoefive** [nss-evening-cohort-14/evening-client-side#56] [Help Me!] [answered]
Has anyone had an issue with accepting an assignment? I have accepted this assignment since this past Tuesday and every time I do this error message comes up. I've clicked on the repo link on the Js-3 ticket and sent me to GitHub Class room site and after I click the accept button I wait then refresh then this error message always pops up. I then look in the https://github.com/nss-evening-cohort-14 and the Js-3 assignment is never populated or shows up. I've been trying to complete this assignment for over two day now. 

Please Help!!

**There was an error accepting the assignment. Starter code template repository was not found. The repository might be deleted, the classroom's organization does not own the repository, or the classroom's organization has restrictions on third-party access.. If this problem persists, please contact support.

repository
You accepted the assignment, JS Practice 3 . We're configuring your repository now. This may take a few minutes to complete. Refresh this page to see updates.**

I then wait over 20mins then REACCEPT the assignment and still this error message pops up again!
I've only had this issue with this specific assignment and all the others I had no issues.

**Comment by drteresavasquez** [comment, marked as answer]
@fimoefive try it again. There seemed to be some issues with the repo.

**Reply by fimoefive** [reply]
Thank you very much for resetting the repo. Works now and I'll start on this assignment after class today.
